docs/content/users/quickstart.md

```diff
@@ -628,6 +628,30 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev restart
     ddev launch /admin
     ```
+    
+## ProcessWire
+
+To get started with [ProcessWire](https://processwire.com/), clone the project repository and navigate to the project directory.
+
+=== "MAIN Branch"
+
+    ```bash
+    mkdir my-processwire-site && cd my-processwire-site
+    curl -LJO https://github.com/processwire/processwire/archive/master.zip
+    unzip ./processwire-master.zip && rm -f processwire-master.zip && mv -f ./processwire-master/{.,}* . ; rm -rf processwire-master
+    ddev config --webserver-type=apache-fpm
```

Comment:
```suggestion
    ddev config --webserver-type=apache-fpm
    ddev composer create --prefer-dist --no-interaction processwire/processwire
```

Most of the currently supported frameworks install via composer, where available. This helps with php and vendor constrainsts.